GA-worksheet.md

```diff
@@ -7,18 +7,18 @@
 
 ##Project #2 Part #1: Header HTML
 1. What is the purpose of the header tag?
-
+The purpose of the header tag is to make sure the blog has an order in it's body and we have a heading inside the page. The header tag is another code for the body, but doesn't indent.
```

Comment:
The header sits within the body (as opposed to the head, which is for metadata) and does give the website order.
More specifically, the header tag actually holds the introductory material for the website, such as a logo and navigational links. Check it out on Buzzfeed.com. The top of the page that lets you know that you're on Buzzfeed is the header (inspect element to clarify).